book.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Kindly Make sure the Rentals can aslo be modified in out side the class, thus is should be in `attr_accessor` rather that in a`ttr_reader`